tests/test_uncrustify/failure.py

```diff
@@ -54,5 +66,17 @@ def __init__(self, expected, actual):
 
     # -------------------------------------------------------------------------
     def __str__(self):
-        return 'Output {!r} does not match expected output {!r}'.format(
+        return 'Output {!r} unexpectedly matches expected output {!r}'.format(
```

Comment:
Ack... this text was meant to appear at L81, no? I think this change was unintended.

Reply:
true, I wonder how this one slipped (shrug)